photon-server/src/main/resources/web/index.html

```diff
@@ -1 +1,15 @@
-<p>UI has not been copied!</p>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" href="./favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Photon Client</title>
```

Comment:
```suggestion
    <title>PhotonVision</title>
```

Reply:
Why rename it? I feel like Photon Client better distinguishes the tab for what it is